draft-barnes-acme.md

```diff
@@ -821,7 +831,7 @@ The choice of which Challenges to offer to a client under which circumstances is
 | Existing valid certs, first use of ACME       | DV + Proof of Possession of previous CA-signed key    |
 | Ongoing ACME usage                            | PoP of previous Authorized key                        |
 | Ongoing ACME usage, lost Authorized key       | DV + PoP of ACME-certified Subject key                |
-| ACME usage, all keys and recovery tokens lost | Recertification by another CA + PoP of that key       |
+| ACME usage, all keys and recovery tokens lost | Proof of legal identity of the site owner             |
```

Comment:
This is odd, since the protocol doesn't provide a way to get such proof, and the main implementation of the protocol (Let's Encrypt) will not have any way to validate legal identity. What was your objection to the old language here?